src/GitVersionTask/IGitVersionTaskExecutor.cs

```diff
@@ -0,0 +1,12 @@
+using GitVersionTask.MsBuild.Tasks;
+
+namespace GitVersionTask
```

Comment:
Wouldn't the namespace `GitVersion.MSBuildTask` or something similar make namespace handling easier when `GitVersionTask` references stuff from `GitVersionCore`?

Reply:
Moved the `GitVersionTask` and `GitVersionTask.MsBuild` under `GitVersion.MSBuildTask` namespace